packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,143 @@
+
+Currently, client-based rate limits apply to the OAuth `/authorize` endpoint on both the Okta Org Authorization server and any Custom Authorization Server. All Custom Authorization Servers share a rate limit. The Org Authorization Server has a separate rate limit.
+
+Each valid request made by a user to this endpoint is counted as one request against the respective [authorization server](/docs/concepts/auth-servers/) rate limit bucket (`/oauth2/{authorizationServerId}/v1` or `/oauth2/v1`). The per minute rate limits on these endpoints apply across an Okta tenant.
+
+For example, company Example.com has 10 OAuth applications running in a production environment. Bob's team is launching a new marketing portal that is a single page OAuth application. Unaware of the rate limits on the `/authorize` endpoint, Bob's team begins running some batch testing scripts against the newly created application that makes hundreds of `/authorize` requests in a single minute. Without the client-based rate limiting framework, the new marketing portal application could potentially consume all of the per minute request limits assigned to Example.okta.com and thereby cause rate-limit violations for the rest of the users that access the other OAuth applications.
+
+Client-based rate limiting can be helpful in the following scenarios:
+
+* You have several OAuth applications that are split across multiple teams. You need to ensure that one OAuth application doesn't consume all of the assigned per minute request limits.
+* You want the ability to isolate any rogue OAuth applications from other valid OAuth applications.
+* You want to ensure that every application team adheres to best-coding practices such as proper error handling without resulting in redirect loops.
+* You want protection against random synthetic tests or batch jobs that make numerous requests every minute in production.
+
+### How it works
+
+The best way to describe how client-based rate limiting works is to provide some example use cases.
+
+#### Client-based isolation for a unique IP address
+
+Bob and Alice are working from home and have distinct IP addresses. Both Bob and Alice try to sign in to their company's portal application (clientID: portal123) at the same time. When they make the authorize request to `https://company.okta.com/oauth2/v1/default/authorize?clientId=portal123`, the client-based rate-limiting framework creates a unique per minute request quota for both Bob and Alice using their IP address and the OAuth client ID that they are trying to access.
+
+Bob: (IP1 + portal123) Gets a quota of 500 total requests per minute and a maximum of two concurrent requests
```

Comment:
Why are these 500? This should be only 60 as mentioned in the draft doc.

Bob: (IP1 + portal123) Gets a quota of 60 total requests per minute and a maximum of 2 concurrent requests
Alice: (IP2 + portal123) Gets a quota of 60 total requests per minute and a maximum of 2 concurrent requests